src/pages/help-centre/components/_questions-template.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { memo } from 'react'
+import styled from 'styled-components'
+import { TQuestionsData } from '../data/_data-types'
+import { useFilteredQuestions } from '../data/_hooks'
+import { getUntranslatedCategory } from './_utility'
+import SideTab from './_side-tab'
+import AnswerCard from './_answer-card'
+import { Community, DidntFindYourAnswerBanner } from './_lazy-load'
+import Layout from 'components/layout/layout'
+import { Localize, localize } from 'components/localization'
+import { StyledLink } from 'components/elements'
+import { Container, SEO } from 'components/containers'
+import { usePlatformQueryParam } from 'components/hooks/use-platform-query-param'
+
+type TQuestionsTemplate = {
+    data: TQuestionsData
+}
+
+const ContactContainer = styled.div`
+    margin-top: 8rem;
+`
+
+const QuestionsTemplate = ({ data }: TQuestionsTemplate) => {
+    const { platform, has_platform } = usePlatformQueryParam()
+    const { questions, category } = data
+    const untranslate_category = getUntranslatedCategory(category)
+    const filtered_questions = useFilteredQuestions(questions)
+
+    return (
+        <Layout>
+            <SEO
+                title={localize(
+                    `_t_Help centre | Frequently asked questions | ${untranslate_category} | Deriv_t_`,
```

Comment:
What is it returning ${untranslate_category}

Reply:
It returns a `string` without **_t_**